src/app-layout/__integ__/awsui-applayout.test.ts

```diff
@@ -32,13 +33,21 @@ class AppLayoutPage extends BasePageObject {
 }
 
 function setupTest(
-  { viewport = viewports.desktop, pageName = 'default', trackResizeObserverErrors = true, visualRefresh = false },
+  {
+    viewport = viewports.desktop,
+    pageName = 'default',
+    trackResizeObserverErrors = true,
+    visualRefresh = false,
+    windowVR = false,
+  },
   testFn: (page: AppLayoutPage) => Promise<void>
 ) {
   return useBrowser(async browser => {
     const page = new AppLayoutPage(browser);
     await page.setWindowSize(viewport);
-    await browser.url(`#/light/app-layout/${pageName}?visualRefresh=${visualRefresh}`);
+    await browser.url(
+      `#/light/app-layout/${pageName}?visualRefresh=${visualRefresh}${windowVR ? '&windowVR=true' : ''}`
```

Comment:
When we have too many query parameters I suggest using `URLSearchParams` to build the query string.  Example in our code: https://github.com/cloudscape-design/components/blob/df04533aa56157985792e1ceb8aaf965a77a193d/src/internal/components/dropdown/__integ__/dropdown-focus-trap.test.ts#L15-L19